src/components/NavBar/NavbarDesktop.jsx

```diff
@@ -67,6 +67,7 @@ const NavbarDesktop = ({ setShowConfirmation }) => {
           <NavbarLinks aria-label="navigation links" />
           <Box sx={{ flexGrow: 1 }} />
           <Box aria-label="menu" sx={{ display: { xs: 'none', md: 'flex' } }}>
+            lhgjasdgdk
```

Comment:
Think we can remove this from the file. Not sure what this would be for in the navbar.